mantidproject/mantid
# Indirect Bayes Quest - Remove Quest code

Remove old code for Quest that has been replaced by (refactored into) `BayesStretch` algorithm. 

**To test:**
- Code review to ensure that the files in question and the QuestRun for IndirectBayes has been removed. 
- Ensure that the ISISIndirectBayes system test passes

Fixes #17091 and #13546

_Does not need to be in the release notes. - This code was not being used other than through the interface Bayes > Stretch_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
